### ISSUE TITLE
Gradient overflow

### ISSUE THREAD

**Issue by RiverMount** [allenai/ms2#6]
Hi！I use your code to achieve text2text task and use the BART configuration, but in the training process

- gradient overflow always occurs in every epoch
- I find all epoches (except the last one) cannot process all the steps, skipping the rest steps at every epoch

![image](https://user-images.githubusercontent.com/43932222/117315463-4e8dd480-aeba-11eb-9969-df21343256a1.png)

If you see, please reply me soon,Thanks!

**Comment by jayded** [comment]
Hi,

Apologies for not seeing this - please tag me directly in any issues. I reproduced these models after installing the `requirements.txt` - it is crucial that the versions specified are the ones we specify.

I ran my models out to 20 epochs, and (1) did not see any major difference in the val losses vs. 8 epochs, and (2) did not observe this behavior. 

If you're still working with this, please verify that you have matched the requirements.txt file and try running to 8 epochs. Or download the models that we've released, see the [ReadMe](https://github.com/allenai/ms2/blob/master/README.md)

Jay

**Comment by jayded** [comment]
Closing - this issue is stale. Please re-open if this becomes active again.
